lib/internal/test_runner/tap_stream.js

```diff
@@ -94,6 +110,11 @@ class TapStream extends Readable {
     this.#tryPush(`${indent}# ${tapEscape(message)}\n`);
   }
 
+  invalid(indent, message) {
+    this.emit('test:invalid', message);
```

Comment:
not sure what this event means? `test:success` and `test:failure` make sense but I don't think this event is needed

Reply:
This is meant to be a new event when reporting invalid TAP output. Should I remove it?